packages/ComponentLibrary/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import modalStyles from './modal.styles';
+
+interface ModalCustomProps {
+  height?: number;
+  width?: number;
+  posX?: number;
+  posY?: number;
+  children: React.ReactNode;
+}
+
+const ModalCustom: React.FC<ModalCustomProps> = ({
```

Comment:
**Caution with this!** If you want to override other styles of the component using the sx prop from **MUI**, you won't be able to do it. Develop the code in such a way that these props can be used in the component so that the client can utilize them if they wish/need to